pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3617,7 +3617,11 @@ func findSecurityGroupForInstance(instance *ec2.Instance, taggedSecurityGroups m
 		// We create instances with one SG
 		// If users create multiple SGs, they must tag one of them as being k8s owned
 		if len(tagged) != 1 {
-			return nil, fmt.Errorf("Multiple tagged security groups found for instance %s; ensure only the k8s security group is tagged", instanceID)
+			taggedGroups := ""
+			for _, v := range tagged {
+				taggedGroups += fmt.Sprintf("%v(%v) ", v.GroupId, v.GroupName)
```

Comment:
I think these are `*string` so you should `fmt.Printf("%s(%s) ", aws.StringValue(v.GroupId), aws.StringValue(v.GroupName))`